tests/blockstore/store.test.ts

```diff
@@ -152,12 +154,19 @@ describe("MetaStore with a saved header", function () {
   });
 
   it("should have a header", async function () {
-    const data = decoder.decode(await raw.get(store.url(), "main"));
-    expect(data).toMatch(/car/);
+    const bytes = await raw.get(store.url(), "main");
+    const data = decoder.decode(bytes);
+    expect(data).toMatch(/parents/);
     const header = JSON.parse(data);
-    expect(header).toBeTruthy();
-    expect(header.cars).toBeTruthy();
-    expect(header.cars[0]["/"]).toEqual(cid.toString());
+    expect(header).toBeDefined();
+    expect(header.parents).toBeDefined();
+
+    const decodedHeader = await (
+      store as unknown as { decodeMetaBlocks: (file: Uint8Array) => Promise<bs.DbMeta> }
+    ).decodeMetaBlocks(bytes);
+    expect(decodedHeader).toBeDefined();
+    expect(decodedHeader.cars).toBeDefined();
+    expect(decodedHeader.cars[0].toString()).toEqual(cid.toString());
```

Comment:
how this is twice as long now... I bet we can flat it to one encoding

Reply:
I want the cid and parents to be in json, maybe we should just bring the cars up to the json also